packages/api/utils/corsConfig.ts

```diff
@@ -0,0 +1,19 @@
+interface Options {
```

Comment:
I like this but I have two questions:
- is this or a similar type already available from the Middy `cors` package?
- if not, can we give this a better name like `CorsHeaders` or something?

Reply:
This is basically the type from the middy `cors` package. I tried to import and use it, but I was not able to, so I copied it.